packages/openneuro-components/src/modal/Modal.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Button } from '../button/Button'
+
+import './modal.scss'
+
+export interface ModalProps {
+  children?: object
+  isOpen?: boolean
+  toggle?: () => void
+  closeText: string
+}
+
+export const Modal: React.FC<ModalProps> = ({
+  children,
+  isOpen,
+  toggle,
+  closeText,
+  ...props
+}) => {
+  const showModal = isOpen ? 'show-modal' : 'hide-modal'
+  return (
+    <div className={'modal-wrapper ' + showModal} {...props}>
+      <div className="overlay" onClick={toggle}></div>
+
+      <div className="grid modal" isOpen={isOpen} toggle={toggle}>
```

Comment:
@thinknoack Looks like isOpen and toggle can just be removed here. They are not valid div properties.

Reply:
*removed